src/dswx_sar/refine_with_bimodality.py

```diff
@@ -1170,50 +1283,73 @@ def fill_gap_water_bimodality_parallel(
 
                     size = sizes[ind]
                     index_set.append(ind)
-                    component_data[ind] = (ind, size, [sub_x_start, sub_x_end, sub_y_start, sub_y_end])
+                    component_data[ind] = (ind,
+                                           size,
+                                           [sub_x_start,
+                                            sub_x_end,
+                                            sub_y_start,
+                                            sub_y_end])
 
             for pol_ind, pol in enumerate(pol_list):
                 if pol in ['VV', 'VH', 'HH', 'HV']:
-                    logger.info(f'filling bright water bodies with bimodality using {pol}')
-                    bimodality_output = np.zeros([len(sizes)], dtype='byte')
-                    check_output = np.ones([len(sizes)], dtype='byte')
+                    logger.info(
+                        'filling bright water bodies with bimodality '
+                        f'using {pol}')
+                    bimodality_output = np.zeros([len(sizes)],
```

Comment:
Here and immediately below - List of scalars only necessary for multi-dimensional zeros/ones arrays. e.g. np.zeros([5, 6])
```suggestion
                    bimodality_output = np.zeros(len(sizes),
```

Reply:
Thanks. @LiangJYu I fixed all having similar issues.